src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Windows.cs

```diff
@@ -78,7 +91,9 @@ public static SocketError CreateSocket(AddressFamily addressFamily, SocketType s
                     return GetLastSocketError();
                 }
 
-                Interop.Winsock.WSAProtocolInfo protocolInfo = Marshal.PtrToStructure<Interop.Winsock.WSAProtocolInfo>((IntPtr)pinnedBuffer);
+                Interop.Kernel32.SetHandleInformation(socket, Interop.Kernel32.HandleFlags.Inherit, 0);
```

Comment:
`SetHandleInformation` returns false on failure.  We should either check for failure and throw, or, if this truly has no effect (per above comment), add a comment here noting why it is not being checked.